<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpScreen</class>
 <widget class="QDialog" name="SignUpScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartShopXpress</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/img0/logo3.png</normaloff>:/img/img0/logo3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/img/img0/signinBackground.png);</string>
  </property>
  <widget class="QLabel" name="signUpLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/signup2.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>136</y>
     <width>278</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>456</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNoLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>349</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mailLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>243</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/menuicon.png);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>500</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/createacc.png);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>296</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="genderComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>402</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <item>
    <property name="text">
     <string>--</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
